test/NonSilo.Tests/SerializationTests/SerializationTests.DifferentTypes.cs

```diff
@@ -131,6 +133,28 @@ public void SerializationTests_RecursiveSerialization()
 
             TestTypeA output2 = this.fixture.SerializationManager.RoundTripSerializationForTesting(input);
         }
+
 #endif
+
+        [Fact, TestCategory("BVT"), TestCategory("Functional"), TestCategory("Serialization")]
+        public void SerializationTests_CultureInfo()
+        {
+            var input = new List<CultureInfo> { CultureInfo.GetCultureInfo("en"), CultureInfo.GetCultureInfo("de") };
+
+            foreach (var cultureInfo in input)
+            {
+                var output = this.fixture.SerializationManager.RoundTripSerializationForTesting(cultureInfo);
+                Assert.Equal(cultureInfo, output);
+            }
+        }
+
+        [Fact, TestCategory("BVT"), TestCategory("Functional"), TestCategory("Serialization")]
+        public void SerializationTests_CultureInfoList()
```

Comment:
Ah, I see. If these tests were put into BuiltInSerializerTests.cs where we disable the fallback serializer, then the first test would be sufficient.